lib/environment_helpers/enumerable_helpers.rb

```diff
@@ -0,0 +1,31 @@
+module EnvironmentHelpers
+  module EnumerableHelpers
+    VALID_TYPES = %i[strings symbols integers file_paths]
+
+    def array(key, of: :strings, delimiter: ",", default: nil, required: false)
+      check_default_type(:array, default, Array)
```

Comment:
Should we also check on the types of the items _in_ the default array?

Reply:
Sure!